### ISSUE TITLE
Joined requests

### ISSUE THREAD

**Issue by robert-cronin** [RxSwiftCommunity/RxGRDB#37]
Does RxGRDB support joined requests? For example, I have 2 structures called Flight and Aircraft as defined below,
```
struct Flight {
    var FLIGHT_ID: String
    var AC_ID: String
    ...
}
```

```
struct Aircraft{
    var AC_ID: String
    var AC_REGISTRATION: String
    ...
}
```
I would like to bind the value of AC_REGISTRATION to a textField based on a given value of FLIGHT_ID.
Is there any way to do this in RxGRDB?

**Comment by groue** [comment]
Hello again @robert-cronin,

Your question is not about RxGRDB, but about [GRDB](http://github.com/groue/GRDB.swift).

You should find the information you need in the [Associations Guide](https://github.com/groue/GRDB.swift/blob/master/Documentation/AssociationsBasics.md), that has just shipped with GRDB 3.0.

In your future questions, it will be appreciated that you show that you have spend a little of your own time looking for answers before requesting help. Fortunately, it happens that RxGRDB and GRDB come with up-to-date documentation:

- GRDB: https://github.com/groue/GRDB.swift/blob/master/README.md#documentation
- RxGRDB: https://github.com/RxSwiftCommunity/RxGRDB/blob/master/README.md#documentation

Your questions will become more precise, and more useful for other users, too.

Thanks for your cooperation.

**Comment by robert-cronin** [comment]
That seemed to do the trick!
I had a look through the Associations Guide but was not able to subclass FetchableRecord or TableRecord. Guess I should have included that in my question!

Thanks for your help Groue, its an awesome package 👍
